src/lmdb-js.h

```diff
@@ -282,10 +282,13 @@ typedef struct callback_holder_t {
 	EnvWrap* ew;
 	std::vector<napi_threadsafe_function> callbacks;
 } callback_holder_t;
+
 typedef struct user_buffer_t {
-	MDB_val buffer;
+	napi_env env;
```

Comment:
This isn't actually used (read) is it? (And I thought it was considered unsafe to store napi_env anyway)

Reply:
It's referenced in the destructor so that the ref can be released: https://github.com/kriszyp/lmdb-js/pull/320/files#diff-f38d85b2982fc3a98c8a1dfe5f1cab0eea8b2e9ad9657f69d3e6338f9a25079dR1079.

Hmm, yeah storing the `napi_env` is probably a bad idea. Might be moot if since I will probably store the value instead of the `napi_ref`.